README.md

```diff
@@ -277,7 +277,7 @@ Writing a new type definition is as simple as creating a new package. Start by c
 
 * **main** The entry point to the definition (canonical to "main" in NPM's `package.json`)
 * **browser** A string or map of paths to override when resolving (following the [browser field specification](https://github.com/defunctzombie/package-browser-field-spec))
-* **ambient** Denote that this definition _must_ be installed as ambient
+* **ambient** Denote that this definition should be installed as ambient to the most likely target environment
```

Comment:
I'm also confused reading this. For me it makes things more opaque and less comprehensible.   

It's difficult expressing things in a way that doesn't hide complexity but remains understandable.